Source/WebKit/_WebKit_SwiftUI/Implementation/WebViewRepresentable.swift

```diff
@@ -113,6 +113,10 @@ struct WebViewRepresentable {
         // issues manifest.
         return CGSize(width: width.rounded(.down), height: height.rounded(.down));
     }
+
+    static func dismantlePlatformView(_ platformView: CocoaWebViewAdapter, coordinator: WebViewCoordinator) {
+        coordinator.configuration.owner.page.isBoundToWebView = false
```

Comment:
Maybe something for the future: I wonder if instead of just a boolean we could use some sort of `@Namespace` value inside the representable which would let us define both owned/unowned while also allowing for us to verify _expected_ ownership via `==`.

Reply:
That's a good idea!